sqldb/test_postgres.go

```diff
@@ -0,0 +1,33 @@
+//go:build test_db_postgres
+// +build test_db_postgres
+
```

Comment:
question: Could you explain why we are naming the test files with the prefix `test` instead of the suffix `test`?

Reply:
yes, comes from [`taproot_assets`](https://github.com/lightninglabs/taproot-assets/blob/main/tapdb/test_postgres.go) but as you noticed it would be good to change it to match the Go's good practices